lib/resources/websockets.js

```diff
@@ -197,7 +197,7 @@ class AlpacaStreamClient extends events.EventEmitter {
   }
 
   subscriptions() {
-    return Object.keys(this.subscriptionState)
+    return Object.keys(this.subscriptionState).filter(x => !!this.subscriptionState[x], this)
```

Comment:
I've added your fix in this commit: https://github.com/alpacahq/alpaca-trade-api-js/commit/4d7112f2ce8e6b6e340bda34c4dce9ef8243ea5c

added a doc string explaining why, and simplified the code a bit (the subscription data is saved with booleans, no need for !!)

Reply:
thank you for your contribution!